### PR TITLE
add prepare step to remove ".cargo/config"

### LINKED ISSUE #650: docs.rs should ignore .cargo/config
Fixing this should be basically the same steps as https://github.com/rust-lang/docs.rs/issues/555#issuecomment-571950211, rustwide should remove `.cargo/config` before starting the build.

_Originally posted by @jyn514 in https://github.com/rust-lang/docs.rs/issues/647#issuecomment-602042380_

### PR DESCRIPTION
**Pull request by Freax13**
fixes: https://github.com/rust-lang/docs.rs/issues/650

### CODE REVIEW

File: src/prepare.rs [hunk 1]
```diff
@@ -68,6 +70,14 @@ impl<'a> Prepare<'a> {
         Ok(())
     }
 
+    fn remove_cargo_config(&self) -> Result<(), Error> {
+        let path = self.source_dir.join(".cargo").join("config");
+        if path.exists() {
+            remove_file(path)?;
```
Comment (emilyalbini) [inline src/prepare.rs]: Can you add an `info!` here that says the `.cargo/config` file was deleted? I'd prefer for all the changes done during the prepare to be logged, so it's easy to see what changed between the local build and the docs.rs build when debugging build differences.

**Review by emilyalbini — CHANGES_REQUESTED** [review]
Thanks for this PR! Other than addressing the comment below, could you add a changelog entry (in a new "Changed" subsection under "Unreleased") for this?

**Review by emilyalbini — APPROVED** [review]

**Comment by emilyalbini** [comment]
Thanks for the PR! I'll release a new rustwide version after merging https://github.com/rust-lang/rustwide/pull/20.

**Comment by emilyalbini** [comment]
Released rustwide 0.7.0! Note that there are breaking changes in it (see the changelog).

**Comment by phil-opp** [comment]
> Released rustwide 0.7.0! Note that there are breaking changes in it (see the changelog).

I think you forgot to add the "0.7.0" heading to the changelog. It still shows the 0.7.0 changes under "unreleased".

**Pull request merged**